src/lib/events/duplicateEvent.ts

```
@@ -0,0 +1,65 @@
+import { SupabaseClient } from "@supabase/supabase-js";
+import { addDays, parseISO } from "date-fns";
+
+/**
+ * Duplicates an event, shifting its start/end/event dates 7 days forward
+ * and creating it as a draft.
+ * 
```

ESLint

[error] Delete `·`

(prettier/prettier)